src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { VNode } from 'snabbdom/vnode';
+import { VNode } from 'snabbdom/build/package/vnode';
```

Comment:
Please import without relying on the package structure. AFAIK `from 'snabbdom'` should work

Reply:
[SnabbDOM 1.x](https://github.com/snabbdom/snabbdom/commit/a9f7e74c2499e36c3e1cb8097f704c08e25c1555) refactors its Package structure, so I make this PR.

Reply:
https://github.com/snabbdom/snabbdom/issues/568
AFAIK the old import should continue to work

Reply:
But it failed on Type definations...